File: Assignment__17/Question5.java
import java.util.Scanner;

class logic
{
    void DisplayTable(int iNo)
    {
        int iCnt = 0;

        for(iCnt = iNo; iCnt <= (iNo*10); iCnt = iCnt + iNo)
        {
            System.out.println(iCnt);
        }
    }
}

class Question5
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the number : ");
        int iValue = sobj.nextInt();

        int iResult = 0;

        logic lobj = new logic();
        lobj.DisplayTable(iValue);

    }    

} 

File: Assignment__26/Question5.c
#include<stdio.h>

void Display(int iNo)
{
    int iCnt = 0;

    for(iCnt = 1; iCnt <= (iNo*2); iCnt++)
    {
        if(iCnt % 2 == 0)
        {
            printf("%d\t",iCnt);
        }
    }
}


int main()
{
    int iValue = 0;
    printf("Enter the number of frequency : \n");
    scanf("%d",&iValue);

    Display(iValue);

    return 0;
}